bridge/README.md

```diff
@@ -125,18 +125,29 @@ CONFIG_PATH="<path_to_bridge_configuration.ini>"
 
 As an alternative to manually installing dependencies and running scripts, you can use Docker.
 
+### Build docker image
+
+```sh
+docker build -t symbolplatform/bridge:1.1 -f Dockerfile --network host ..
+```
+
 ### Running the Bridge
 
 ```sh
-docker run -d -it --name xym_wxym_bridge --restart always -v $(pwd):$(pwd) symbolplatform/bridge:1.0 /home/ubuntu/product/bridge/xym_wxym_bridge/run-bridge-docker.sh
+docker run -d -it --name xym_wxym_bridge --restart always -v $(pwd):/data symbolplatform/bridge:1.1 <mode>
 ```
 
-**Note**: `$(pwd):$(pwd)` mounts the current working directory from the host into the same path in the container.
+The `<mode>` must be one of the following:
+* `api` - Starts the API. You must also add `-p 5000:5000` to expose the API port.
+* `wrappedflow` - Runs the wrapped flow.
+* `nativeflow` - Runs the native flow and skips `--unwrap` operations.
 
-### Exposing the API
+**Note**: `$(pwd):/data` mounts the current working directory from the host into the `/data` directory inside the container.
+
+By default, the bridge configuration file is located at `/data/config/configuration.ini`. You can override this path by setting the `BRIDGE_CONFIG_PATH` environment variable.
```

Comment:
```suggestion
By default, the bridge looks for its configuration file in `/data/config/configuration.ini`. You can override this path by setting the `BRIDGE_CONFIG_PATH` environment variable.
```

It would be good to provide an example, because it's not clear if you can override only the path or also the filename.